HLTrigger/JetMET/plugins/BuildFile.xml

```diff
@@ -14,6 +14,7 @@
   <use   name="FWCore/Utilities"/>
   <use   name="RecoMET/METAlgorithms"/>
   <use   name="HLTrigger/HLTcore"/>
+  <use   name="HLTrigger/JetMET"/>
```

Comment:
since the src directory is full of plugins rather than a library, I believe this is not allowed. Why is it needed? [of course one should also ask why this package needs two directories of plugins, but thats a pre-existing problem]